GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -8,10 +8,13 @@ namespace GVFS.Platform.Mac
 {
     public partial class MacPlatform : POSIXPlatform
     {
-        public MacPlatform()
+        public MacPlatform() : base (
+             underConstruction: new UnderConstructionFlags(
+                supportsGVFSUpgrade: false,
+                supportsGVFSConfig: true))

```

Comment:
With this change can the `ConfigVerbTests` functional tests be turned on for Mac?